dashboard/app/auth_test.go

```diff
@@ -1,3 +1,6 @@
+// Copyright 2017 syzkaller project authors. All rights reserved.
```

Comment:
s/2017/2021/
and can be moved to the previous commit (or do you plan to squash?)